src/components/calcite-block/calcite-block.tsx

```diff
@@ -156,12 +180,19 @@ export class CalciteBlock {
     const hasContent = !!Array.from(el.children).some((child) => child.slot !== CONTROL_SLOT_NAME);
 
     return (
-      <article aria-expanded={collapsible ? (open ? "true" : "false") : null}>
-        {headerNode}
-        <div class={CSS.content} hidden={!hasContent || !open}>
-          <slot />
-        </div>
-      </article>
+      <Host
+        aria-expanded={collapsible ? (open ? "true" : "false") : null}
+        aria-busy={loading}
+        aria-disabled={disabled}
```

Comment:
does this need the strings "true" "false" like aria-expanded?

Reply:
Same with aria-busy

Reply:
`aria-expanded` needs strings because there's a different meaning when not present:

👇from https://www.w3.org/TR/wai-aria-1.0/states_and_properties#aria-expanded 

Value | Description
-- | --
true | The element, or another grouping element it controls, is expanded.
false | The element, or another grouping element it controls, is collapsed.
undefined (default) | The element, or another grouping element it controls, is neither expandable nor collapsible; all its child elements are shown or there are no child elements.

A boolean for `aria-busy` is OK because the [default is false](https://www.w3.org/TR/wai-aria-1.0/states_and_properties#aria-busy).

I think we can drop `aria-disabled` since the `disabled` attribute is reflected and has semantic meaning.

Reply:
[fixme] BTW, this is reverting https://github.com/Esri/calcite-app-components/commit/58a8f85b81db3362cf90f58bc944e97721200370#diff-5143375a6c30f688a32734e78d4fb3bc.

Reply:
So did I need to do anything here other than remove `aria-disabled`?

Reply:
That and moving the attributes back to the `<article>` element (see the diff above).

Reply:
COOL.

Reply:
Also, to be clear, we don't need `aria-disabled` for accessibility because the `disabled` attribute does the job.

Reply:
Please have another peek.